src/scss/components/_initiatives.scss

```diff
@@ -1,16 +1,218 @@
+.workshop-image {
+	max-width: 100%;
+}
+
+.comments {
+	margin-top: rem(50);
+
+	p:first-child {
```

Comment:
This selector to set the first comment to grey light is not useful as line 17-23 also sets the same for the submitted comment.